src/app/modules/main/chat_section/chat_content/messages/view.nim

```diff
@@ -172,8 +172,7 @@ QtObject:
     self.messageSearchOngoingChanged()
 
   proc addNewMessagesMarker*(self: View) {.slot.} =
-    if self.model.newMessagesMarkerIndex() == -1:
-      self.delegate.resetNewMessagesMarker()
+    self.delegate.resetNewMessagesMarker()
```

Comment:
It will break the behavior of new messages marker in a certain case, that is:
1) open a chat with a new messages marker:
```
.
.
.
10
----- new messages ----
11
12
```

2) scroll up, so that `d.isMostRecentMessageInViewport === false`
3) receive message on that chat and scroll down
4) what will happen is:
```
.
.
.
10
11
12
----- new messages ----
13
```

what is expected is:
```
.
.
.
10
----- new messages ----
11
12
13
```
New messages marker should never change its position once we enter the chat.

Reply:
fixed